04-trade-protocol.md

```diff
@@ -113,6 +113,7 @@ message TradeProposeRequest {
 message TradeProposeReply {
   SwapAccept swap_accept = 1;
   SwapFail swap_fail = 2;
+  int64 expiry_time_unix = 3;
```

Comment:
We would likely define this field as a `uint64` since it can't assume negative values.

The `uint64` type is a variable length encoding, if we want this field to always be 8 bytes long, we should use `fixed64` (check out the [doc](https://developers.google.com/protocol-buffers/docs/proto3#scalar))

Reply:
uint64 is good